.github/workflows/build-and-push-image.yml

```diff
@@ -115,14 +115,33 @@ jobs:
       - name: Npm install
         run: npm install
 
-      - name: Run cypress
-        run: npx cypress run --env url='${{ secrets.AZURE_ENDPOINT }}',authorizationHeader='${{ secrets.AUTH_HEADER_CYPRESS }}'
+      - name: Run cypress (dev)
+        run: npm run cy:run -- --env url='${{ secrets.AZURE_ENDPOINT }}',authorizationHeader='${{ secrets.AUTH_HEADER_CYPRESS }}, academisationApiUrl=${{secrets.CYPRESS_ACADEMISATION_API_URL}}, academisationApiKey=${{ secrets.CYPRESS_ACADEMISATION_API_KEY}}'
         env:
           db: ${{ secrets.DB_CONNECTION_STRING }}
 
       - name: Upload screenshots
         if: ${{ failure() }}
         uses: actions/upload-artifact@v4
         with:
-          name: screenshots-${{ needs.set-env.outputs.environment }}
+          name: screenshots-dev
```

Comment:
Why is this dev and not the envronment? i.e., Test?

Reply:
I just fixed that haha!